.circleci/config.yml

```diff
@@ -25,8 +25,15 @@ executors:
           CXX: g++
           BUILD_FLAGS: -j
           CTEST_FLAGS: -j4 --output-on-failure
+  clang9:
```

Comment:
This is an aside, and not relevant to this patch: I've been thinking we should compile *most* of our tests against the latest version of clang and gcc, and pick just one test to run against our oldest supported version (right now clang6 and gcc7?). The newer versions tend to be faster and the two of us generate a whole lot of CI runs when we're fired up :)

Reply:
I'd be fine with this, but it seems like a distinct issue. Also, it can be a problem because I use CI to support the claims made in our requirements section. If we do not test with GCC7, how can we tell people that we support it? I know that you are concerned with the speed of the tests... so am I. All I am saying is that it is complicated. Let us deal with this in a distinct issue.

Reply:
Yeah, I'll file one. My thought was to have one test with gcc7, just not test all the *variants* there.